ops/ansible/roles/bfd-server/test/pre-test.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
```

Comment:
thanks for the review. Given that this script was pre-existing, is a script only for testing that seems to be working so far, and likely is just one of several for this ansible test that will fail shellcheck, I am not planning on addressing it with this PR. I do agree though with the sentiment and I think we should run it and fix all of the scripts, test or otherwise.